loadable_apps/loadable_sample/Kconfig

```diff
@@ -107,4 +107,15 @@ config EXAMPLES_MICOM_TIMER_INTERVAL
 endif #EXAMPLES_MICOM_TIMER_TEST
 endchoice
 endif #EXAMPLES_LOADABLE_AUTOMATIC_TEST
+
+config WIFI_PLATFORM_LIB
+	bool "Support for Wifi platform library"
+	default n
+	depends on SUPPORT_COMMON_BINARY
+	---help---
+		Support for Wifi platform library. Wifi platform library
+		will contain wifi platform specific code and will included
+		in the common binary. This is done to allow us to perform
```

Comment:
wifi app binary and wifi platform lib binary has dependency for update. Isn't it?

Reply:
Yes. There might be a dependency. 
But this config is just to enable the wifi pf library support. Else all code of wifi app and pf will be built as a single binary. 
Do you think we need to add a dependency here?

Reply:
No. I mean the explanation can be confusing. Because explanation said we can update the wifi app and pf independently.

Reply:
Oh ok. 
But we have made a separate library for wifi pf because we would like to update it separately. 
When we update wifi pf library, the app also needs to be reloaded, because the symbol addresses in the pf will change. 
But, the app does not need to be updated always. An app update is required only if some interface change happens.

Reply:
Yes, I know. Could you add the description about this also? Then it is much better to understand.